app/models/concerns/foreman_remote_execution/orchestration/ssh.rb

```diff
@@ -0,0 +1,60 @@
+module ForemanRemoteExecution
+  module Orchestration::SSH
+    extend ActiveSupport::Concern
+
+    included do
+      before_destroy :ssh_destroy
+      after_validation :queue_ssh_destroy
+      register_rebuild(:queue_ssh_destroy, N_("SSH_#{self.to_s.split('::').first}"))
+    end
+
+    def drop_from_known_hosts(args)
+      proxy_id, target = args
+      proxy = ::SmartProxy.find(proxy_id)
+      begin
+        proxy.drop_host_from_known_hosts(target)
+      rescue => e
+        Rails.logger.warn e.message
+        return false
+      end
+      true
+    end
+
+    def ssh_destroy
+      logger.debug "Scheduling SSH known_hosts cleanup"
+
+      host, _kind, target = host_kind_target
+      proxies = %w(SSH Ansible).map { |kind| host.remote_execution_proxies(kind).values }
+      proxies.flatten.uniq.each do |proxy|
+        queue.create(id: queue_id(proxy.id), name: _("Remove SSH known hosts for %s") % self,
+          priority: 200, action: [self, :drop_from_known_hosts, [proxy.id, target]])
+      end
+    end
+
+    def queue_ssh_destroy
+      return unless should_drop_from_known_hosts?
+
+      ssh_destroy
+    end
+
+    def should_drop_from_known_hosts?
+      host, = host_kind_target
```

Comment:
```suggestion
      host, _, _ = host_kind_target
```
seems bit more readable as it states what we are dropping.

Reply:
:police_officer: doesn't like this
```
C: Style/TrailingUnderscoreVariable: Do not use trailing _s in parallel assignment. Prefer host, = host_kind_target.
      host, _, _ = host_kind_target
```